azext_iot/operations/_mqtt.py

```diff
@@ -77,6 +79,20 @@ def execute(self, data, properties={}, publish_delay=2, msg_count=100):
         from tqdm import tqdm
         try:
             for _ in tqdm(range(msg_count), desc='Device simulation in progress'):
+                device_twin = _iot_device_twin_show(self.target, self.device_id)
```

Comment:
This should be refactored in a way that is more efficient/the intended usage pattern for the device client to receive desired properties. Its not efficient that for every d2c message we are making an API request to the service for the twin.